docs: add mcp + cli webinar callouts (#77191)
metabase/metabase: docs/ai/mcp.md

Metabase includes an [MCP (Model Context Protocol)](https://modelcontextprotocol.io/) server that lets AI clients connect directly to your Metabase, all scoped to the connecting person's permissions.

> > To learn about using the Metabase MCP server and CLI for AI data exploration, [register for the MCP and CLI workshop](https://www.metabase.com/events/workshop-metabase-mcp-cli).

Reviewer: ```suggestion
> To learn how to use the Metabase MCP server and CLI for AI data exploration, [register for our free MCP and CLI workshop](https://www.metabase.com/events/workshop-metabase-mcp-cli).
```
